readthedocs/projects/models.py

```diff
@@ -898,3 +905,36 @@ class WebHook(Notification):
 
     def __unicode__(self):
         return self.url
+
+
+class Domain(models.Model):
+    project = models.ForeignKey(Project, related_name='domains')
+    objects = RelatedProjectManager()
```

Comment:
The manager object should be placed between the fields and the Meta class as this is the most common place to put it (and shown in the Django docs).